test/runtests.jl

```diff
@@ -25,4 +26,18 @@ using Test
         @test nwscore(nothing, base; gap = -2) == -2
     end
 
+    # Test error thrown by double gap
+    for (b1, b2) in combinations(bases, 2)
+        @test_throws ArgumentError nwscore(nothing, nothing)
+        @test_throws ArgumentError nwscore(nothing, nothing; match = 2, mismatch = -1, gap = -3)
+        @test_throws ArgumentError nwscore(nothing, nothing; mismatch = -4, gap = -3)
+        @test_throws ArgumentError nwscore(nothing, nothing; match = 1, gap = 0)
+    end
+
+    # Test nwaligner()
+    sampleString = "AGCTGTGA"
+    sampleString2 = "ACTGTGG"
+    @test nwaligner(sampleString, sampleString2) isa Array
```

Comment:
This could be more specific, probably. Maybe `... isa `Array{Array{Char,1}, 1}`

Reply:
what do the 1's refer to?

Reply:
The dimension - a vector is a 1 dimensional array, so `Vector{T}` is an alias for `Array{T,1}`. And actually, a matrix is a 2D array, so `Matrix{T}` is an alias for `Array{T,2}`